components/VM_Arm/src/main.c

```diff
@@ -445,8 +454,7 @@ static int vmm_init(void)
         cspacepath_t path;
         vka_cspace_make_path(vka, cap, &path);
         int utType = device ? ALLOCMAN_UT_DEV : ALLOCMAN_UT_KERNEL;
-        if (utType == ALLOCMAN_UT_DEV &&
-            paddr >= ram_paddr_base && paddr <= (ram_paddr_base + (ram_size - 1))) {
+        if (utType == ALLOCMAN_UT_DEV && is_paddr_in_vm_ram(paddr)) {
```

Comment:
How about just `paddr >= ram_paddr_base && paddr < ram_paddr_base + ram_size`?

Reply:
That is also an option. Having a helper function avoids some redundancy. This comes handy in a few other PR also.